package.json

```diff
@@ -5,8 +5,14 @@
   "main": "index.js",
   "scripts": {
     "start": "serve .",
-    "test": "eslint .",
-    "fix": "eslint . --fix"
+    "test": "jest --coverage -w =100%",
```

Comment:
What's the meaning of `=100%`? For setting a coverage threshold you will need to add it on the Jest configuration: https://jestjs.io/docs/en/configuration#coveragethreshold-object.

Reply:
`-w` refers to `workers` in the sense of how many Cores from your CPU Jest can use for Run the Unit Tests. In this case, it will use all the cores. [Docs](https://jestjs.io/docs/en/cli.html#--maxworkersnumstring)

Reply:
Awesome, didn't know about that. And got confused thinking `-w` was for `--watch`. Thanks!